nx/include/switch/applets/error.h

```diff
@@ -0,0 +1,74 @@
+/**
+ * @file error.h
+ * @brief Wrapper for using the error LibraryApplet.
+ * @author StuntHacks
+ * @copyright libnx Authors
+ */
+#pragma once
+#include "../types.h"
+#include "../services/applet.h"
+
+/// Error configuration struct.
+typedef struct {
+    u32 major_code;             ///< First part of the error code.
```

Comment:
error module/description

Reply:
I'm unsure what you mean, could you elaborate?

Reply:
https://switchbrew.org/wiki/Error_codes